clang/test/CodeGen/amdgpu-buffer-rsrc-type-debug-info.c

```diff
@@ -0,0 +1,9 @@
+
+// REQUIRES: amdgpu-registered-target
+// RUN: %clang_cc1 -triple amdgcn -emit-llvm -o - %s -debug-info-kind=limited 2>&1 | FileCheck %s
+
+// CHECK: name: "__amdgcn_buffer_rsrc_t",{{.*}}baseType: ![[BT:[0-9]+]]
+// CHECK: [[BT]] = !DICompositeType(tag: DW_TAG_structure_type, name: "__amdgcn_buffer_rsrc_t", {{.*}} flags: DIFlagFwdDecl)
```

Comment:
For the debug type, I made it a struct type, because I don't think we expose it as a pointer even though it is used internally as one. It is not an integer as well.

Reply:
Yeah, debug as a struct's probably a more sensible representation, since it basically is a struct